README.adoc

```diff
@@ -12,3 +12,34 @@ The `xorg.conf` file should be placed in `/etc/X11/`.
 The `config.txt` file is the equivalent of a BIOS on the pi. It contains system configuration parameters. 
 This file is needed on the primary and secondary pi.
 The `config.txt` file should be placed in `/boot/`.
+
+= Instructions to run SetupScript
+
+On the raspberry pi(rpi), run rasp-config, and increase the RAM to 512MB.
+----
+sudo rasp-config
+----
+(rpi)Ensure your raspberry pi is up-to-date:
```

Comment:
The wiki linked in the issue was a wiki for cross compiling on the pi, so it was split to pc & rpi commands. We're compiling natively, so we don't need to differentiate with `(rpi)`